packages/connect-web/src/index.ts

```diff
@@ -55,9 +56,14 @@ export class TrezorConnectDynamicImpl implements ConnectFactoryDependencies {
     }
 
     public async init(settings: Partial<ConnectSettings> = {}) {
-        if (settings.useCoreInPopup || settings.coreMode === 'popup') {
+        const env = settings.env || getEnv();
+        if (settings.coreMode === 'iframe' || settings.popup === false || env === 'webextension') {
+            this.currentTarget = 'iframe';
+        } else if (settings.useCoreInPopup || settings.coreMode === 'popup') {
```

Comment:
`useCoreInPopup` is now deprecated but used in production right? I am just thinking - could we add a console.warn if this variable is set?

Reply:
Ok actually it shouldn't be deprecated, it should remain but as internal. I just did a refactor to separate the public/internal settings and I removed it from the init method. It was never publicly documented, so it should be safe